config/root/clusterworkspacetype-organization.yaml

```diff
@@ -1,7 +1,11 @@
-apiVersion: tenancy.kcp.dev/v1alpha1
-kind: ClusterWorkspaceType
-metadata:
-  name: organization
-spec:
-  initializers:
-  - tenancy.kcp.dev/organization
+  apiVersion: tenancy.kcp.dev/v1alpha1
```

Comment:
(no action required) why did this file get indented?

Reply:
fixed, oops?